packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/components/ContenTypesSection/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@strapi/design-system/Box';
+import CollapsableContentType from '../CollapsableContentType';
+
+const ContentTypesSection = ({ section, ...props }) => {
```

Comment:
we don't have this yet, I don't think we need this component, or any component related to the granular permissions

Reply:
you are right, we have removed the component